index.html

```diff
@@ -20,11 +21,27 @@ <h1 id="logo">Aziz Mejri</h1>
           <li><a class="nav-link" href="#about-me">About</a></li>
           <li>
             <a class="nav-link" href="#contact"
-              >Contact <i class="far fa-envelope"></i
+              >Contact <i class="fas fa-envelope"></i
             ></a>
           </li>
         </ul>
       </nav>
+      <nav id="mobile-nav">
+        <ul class="nav-menu">
+          <li class="nav-item">

```

Comment:
Kindly place the hamburger icon in its right point, you can do this by adding some margin. And please stick to the figma design.
Your design:
![Screenshot from 2022-04-15 23-13-36](https://user-images.githubusercontent.com/79506105/163622596-04f4e534-31a0-4f82-879d-7de8b89149a8.png)
Figma design

![Screenshot from 2022-04-15 23-12-26](https://user-images.githubusercontent.com/79506105/163622643-9286c0cd-90cf-4b35-bc30-4c36f8f7eefa.png)